docs/en/ingest-management/release-notes/release-notes-8.4.asciidoc

```diff
@@ -27,6 +27,47 @@ Also see:
 [[release-notes-8.4.1]]
 == {fleet} and {agent} 8.4.1
 
+[discrete]
+[[known-issues-8.4.1]]
+=== Known issues
+
+// tag::credentials-error[]
+.Credentials error prevents {agent} from ingesting AWS logs
+[%collapsible]
+====
+
+*Details*
+
+{agent}s configured to use the AWS S3 and SQS integrations may return the
+following error:
+
+[source,shell]
+----
+sqs ReceiveMessage failed: operation error SQS: ReceiveMessage, https response
```

Comment:
beware that the error message might differ for some users: it's anyway most of the time related to authentication and/or region